TileFormats/Instanced3DModel/README.md

```diff
@@ -6,24 +6,55 @@
+    * [Instance scaling](#instance-scaling)
+    * [Examples](#examples)
+        * [Positions only](#positions-only) 
+        * [Quantized positions and oct-encoded normals](#quantized-positions-and-oct-encoded-normals)   
+* [Batch Table](#batch-table)
+* [glTF](#gltf)                            
+* [File extension and MIME type](#file-extension-and-mime-type)
+* [Implementation examples](#implementation-examples)
+    * [Cesium](#cesium)
+
 ## Overview
 
-_Instanced 3D Model_ is a tile format for efficient streaming and rendering of a large number of models, called _instances_, with slight variations.  In the simplest case, the same tree model, for example, may be located - or _instanced_ - in several places.  Each instance references the same model, and has per-instance properties, such as position.  Using the core 3D Tiles spec language, each instance is a _feature_.
+_Instanced 3D Model_ is a tile format for efficient streaming and rendering of a large number of models, called _instances_, with slight variations.  In the simplest case, the same tree model, for example, may be located&mdash;or _instanced_&mdash;in several places.  Each instance references the same model and has per-instance properties, such as position.  Using the core 3D Tiles spec language, each instance is a _feature_.
 
-In addition to trees, Instanced 3D Model is useful for exterior features such as fire hydrants, sewer caps, lamps, and traffic lights, and interior CAD features such as bolts, valves, and electric outlets.
+In addition to trees, Instanced 3D Model is useful for exterior features such as fire hydrants, sewer caps, lamps, and traffic lights, and for interior CAD features such as bolts, valves, and electrical outlets.
 
 A [Composite](../Composite/README.md) tile can be used to create tiles with different types of instanced models, e.g., trees and traffic lights.
 
 Instanced 3D Model maps well to the [ANGLE_instanced_arrays](https://www.khronos.org/registry/webgl/extensions/ANGLE_instanced_arrays/) extension for efficient rendering with WebGL.
 
 ## Layout
 
-A tile is composed of a header section immediately followed by a body section.
-
-**Figure 1**: Instanced 3D Model layout (dashes indicate optional fields).
+A tile is composed of a header section immediately followed by a body section. The following figure shows the Instanced 3D Model layout (dashes indicate optional fields):
 
 ![header layout](figures/header-layout.png)
 
+### Padding
+
+A tile's `byteLength` must be aligned to an 8-byte boundary.
+
+The [binary glTF](#gltf) (if present) must start and end on an 8-byte alignment so that glTF's byte-alignment guarantees are met. This can be done by padding the [Feature Table](../FeatureTable/README.md#padding) or [Batch Table](../BatchTable/README.md#padding) if they are present.
+
+Otherwise, if the glTF field is a UTF-8 string, it must be padded with trailing Space characters (`0x20`) to satisfy alignment requirements of the tile.
```

Comment:
Mention that these trailing spaces should be removed at runtime before requesting the glTF.